### PR TITLE
python: added README.md file for Python bindings.

### PR DESCRIPTION
**Pull request by cristi-iacob**
As reported in this issue #479, I have added a README.md file which contains information to be displayed on the main Pypi libiio page.

### CODE REVIEW

**Comment by cristi-iacob** [comment]
I am not sure what to write in the README file for the bindings. For the moment I have copied the general information about libiio from the original README file, without the download section.

**Comment by rgetz** [comment]
I'm not sure that make 100% sense - I would talk less about libiio, and more about the python bindings.

**Comment by tfcollins** [comment]
I couldn't really make a PR to your repo, so I just put it in a gist: https://gist.github.com/tfcollins/44e47e09dcbc5d4c6cd76eca986c73ba

**Comment by cristi-iacob** [comment]
I have modified the content from the README file with the one @tfcollins suggested. Thanks!

File: bindings/python/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+![open bugs](https://img.shields.io/github/issues/analogdevicesinc/libiio.svg)
+
+[[Docs](https://analogdevicesinc.github.io/libiio/v0.19/python/index.html)]
+[[Support](http://ez.analog.com)]
+[[Wiki](https://wiki.analog.com/resources/tools-software/linux-software/libiio)]
+
+### Requirements
+To use these bindings naturally you need the core library they depend upon, libiio. This is not packaged with the pypi release but there are a number of options:
+  - If you want to just use libiio, we suggest using the [latest release](https://github.com/analogdevicesinc/libiio/releases/latest).
+  - If you think you have found a bug in the release, or need a feature which isn't in the release, try the latest **untested** binaries from the master branch and check out the [documentation](https://codedocs.xyz/analogdevicesinc/libiio/) based on the master branch. We provide builds for a few operating systems. If you need something else, we can most likely add that -- just ask.
```
Comment (rgetz) [inline bindings/python/README.md:18]: https://codedocs.xyz/analogdevicesinc/libiio/ isn't the latest - point to github pages.

File: bindings/python/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Installing the bindings
+To install these bindings there are a few methods. If you already have the library itself and just need the bindings, pip is the most convenient method:
+```
+(sudo) pip install libiio
+```
+If you do not want to use pip, then installation is dependent on your operating system.
+#### Linux / macOS
+For Linux and macOS the python bindings need to be installed through source if not using pip. For v0.20 and beyond this requires the `-DPYTHON_BINDINGS=ON` flag during the cmake configuration. Further documentation is located [here](https://github.com/analogdevicesinc/libiio/blob/master/README_BUILD.md).
```
Comment (rgetz) [inline bindings/python/README.md:27]: It wasn't 0.20, it was 0.19

The change was made Fri Jan 17 (f1faa2816c076eaea285ac4683fefa1c4f5878c9) 
0.19 was released Feb 14 (5f5af2e417129ad8f4e05fc5c1b730f0694dca12)

**Comment by rgetz** [comment]
This is much better - thanks to both.

File: bindings/python/README.md (hunk 1 above)
Comment (cristi-iacob) [reply bindings/python/README.md]: Solved.

File: bindings/python/README.md (hunk 2 above)
Comment (cristi-iacob) [reply bindings/python/README.md]: Solved.

**Comment by rgetz** [comment]
[fenced-code-flag] Missing code language flag (line 22) - see the README_BUILD.md

[heading-increment] Heading levels should increment by one level at a time (line 15)

**Comment by cristi-iacob** [comment]
I have solved this two issues.

**Comment by rgetz** [comment]
merged

**Pull request merged**